src/Raven.Client/Documents/Subscriptions/SubscriptionState.cs

```diff
@@ -22,7 +22,9 @@ public class SubscriptionState : IDatabaseTask, IDatabaseTaskStatus
         public string NodeTag { get; set; }
         public DateTime? LastBatchAckTime { get; set; }  // Last time server made some progress with the subscriptions docs  
         public DateTime? LastClientConnectionTime { get; set; } // Last time any client has connected to server (connection dead or alive)
-        
+        // raft index used to create or update subscription task
+        public long LastModifiedIndex { get; set; }
```

Comment:
usually we call this 'RaftCommandIndex' in the Client API

Reply:
done